source/projects/private-voice-assistant/voice_remote_expose_devices.markdown

```diff
@@ -0,0 +1,15 @@
+---
+title: "Expose devices"
```

Comment:
This should not be under projects. This is generic documentation that applies to Assist, Google and Amazon. I suggest this is moved to it's own PR.